types/node/index.d.ts

```diff
@@ -783,7 +791,9 @@ declare namespace NodeJS {
         chdir(directory: string): void;
         cwd(): string;
         debugPort: number;
+        emitWarning(warning: string | Error, options?: WarningOptions): void;
```

Comment:
I think {options} should be mandatory here as the variant with only one arg is already covered by another overload.